### PR TITLE
Sync Serverless CI

### PR DESCRIPTION
**Pull request by serverless-qe**
Sync Serverless CI using openshift-knative/hack.

### CODE REVIEW

**Comment by creydr** [comment]
```
oc apply -f /go/src/github.com/openshift-knative/eventing/openshift/tls/issuers/eventing-ca-issuer.yaml -f /go/src/github.com/openshift-knative/eventing/openshift/tls/issuers/selfsigned-issuer.yaml
the path "/go/src/github.com/openshift-knative/eventing/openshift/tls/issuers/eventing-ca-issuer.yaml" does not exist
the path "/go/src/github.com/openshift-knative/eventing/openshift/tls/issuers/selfsigned-issuer.yaml" does not exist
``` from [knative-eventing-release-v1.13-411-test-reconciler-aws-ocp-411](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/48159/rehearse-48159-pull-ci-openshift-knative-eventing-release-v1.13-411-test-reconciler-aws-ocp-411/1752025641051492352)

@pierDipi I see the `tls/issuers` folder only in the 1.11 branch and was brought via the trustmanager integration. Don't we need it in 1.12 and 1.13 too?

**Comment by pierDipi** [comment]
`openshift/tls` created on 1.13

**Comment by pierDipi** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-encryp-9732689-aws-ocp-411

**Comment by pierDipi** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-reconciler-aws-ocp-411

**Comment by pierDipi** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-conformance-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-encryp-9732689-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-reconciler-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-414-test-conformance-aws-ocp-414
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-reconciler-aws-ocp-411
/pj-rehearse pull-ci-openshift-knative-eventing-istio-release-next-414-e2e-tests-aws-ocp-414
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-encryp-9732689-aws-ocp-411
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-414-test-e2e-aws-ocp-414
/pj-rehearse pull-ci-openshift-knative-eventing-istio-release-next-411-e2e-tests-aws-ocp-411
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-e2e-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-414-test-conformance-aws-ocp-414

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-reconciler-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-istio-release-next-414-e2e-tests-aws-ocp-414

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-encryp-9732689-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-414-test-e2e-aws-ocp-414

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-istio-release-next-411-e2e-tests-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-e2e-aws-ocp-411

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.13-411-test-reconciler-aws-ocp-411

File: ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-periodics.yaml [hunk 1]
```diff
@@ -79,86 +79,6 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
-- agent: kubernetes
```
Comment (skonto) [inline ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-periodics.yaml:82]: cc @ReToCode

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

File: ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-periodics.yaml (hunk 1 above)
Comment (ReToCode) [reply ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-periodics.yaml:82]: Yes we need to create branches and CI for 1.13 as well. But we need to fix the linked issue first.

**Comment by pierDipi** [comment]
@creydr do you think we can merge this event though there are flakes we could resolve on the actual branches ?

**Comment by creydr** [comment]
> @creydr do you think we can merge this event though there are flakes we could resolve on the actual branches ?

I think so too :+1: 

/unhold

**Pull request merged**